src/components/previewers/VideoPlayer.jsx

```
@@ -0,0 +1,170 @@
+  return (
+    <Previewer isOpen={isOpen} onClose={onClose}>
+      <div className="viewer-container relative">
+        <IoClose
+          className="h-7 w-7 cursor-pointer hover:text-red-500 absolute top-5 right-3 bg-white dark:bg-dark_bg_4 rounded-full p-1 md:bg-transparent md:p-0"
+          onClick={onClose}
+        />
+        <div className="grid grid-cols-1 md:grid-cols-2 pr-3">
+          {/* Image Section */}
+          <div className="flex-1">
+            <div className="player">
+              <video src={video} controls className="w-full h-full" />
```

Biome (1.9.4)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)